ibl-frontend/frontend-content/src/app/session-list/session-list.component.ts

```diff
@@ -375,6 +391,10 @@ export class SessionListComponent implements OnInit, OnDestroy {
                 continue
               }
             }
+            if(key == 'brain_regions' && value !== null){
```

Comment:
There appears to be a small bug related to 'unsetting' the brain filter after use. For instance, when I apply `HPF` as a filter, it appropriately restricts to 2 records but when I simply empty the text box, it shows 0 results.

Reply:
This been address in the latest commit!